articles/connections/database/custom-db/templates/index.md

```diff
@@ -25,15 +25,13 @@ While Auth0 has populated default templates in the Dashboard script editor, you
 Use the following checklist to make sure your scripts achieve the results you intend:
 
 1. **Set a `user_id` on the returned user profile that is consistent for the same user every time.**
-   In the migration scenario, this is important because if you try to set a random `user_id` for each login and you call `forgot password` twice, and then try to follow the second link, the user is unable to migrate. One of the user stores in Mongo gets confused if you have the same email address but more than one ID if you do the forgot password twice. The result is that every time a user logs in, they will get a new user created in the user list. 
-
-   In the non-migration scenario, this is important because you can end up with duplicate users in the Auth0 database.
+   In the migration scenario, this is important because if you try to set a random `user_id` for each login and you call `forgot password` twice, and then try to follow the second link, the user is unable to migrate. In the non-migration scenario, this is important because you can end up with duplicate users in the Auth0 database.
```

Comment:
Sorry, with more testing I've found you don't have to call forgot password twice, just once, so should just be:
```
In the migration scenario, this is important because if you try to set a random `user_id` in the get_user script and you call `forgot password`, and then change the password, the user will get duplicated every time they log in.  In the non-migration scenario if you set a random user_id you can end up with duplicate users for every login.
```

Reply:
Made this change with a slight modification: `In the migration scenario, this is important because if you set a random `user_id` in the `get_user` script, then call `forgot password` and change the password, the user will get duplicated every time they log in.  In the non-migration scenario, if you set a random `user_id` you can end up with duplicate users for every login.`